### PR TITLE
[docs] There is no 'strings' type

### PR DESCRIPTION
**Pull request by FeiyangTan**
no such type "strings", only "string".

### CODE REVIEW

File: docs/types/reference-types.rst [hunk 1]
```diff
@@ -132,11 +132,11 @@ a reference to it.
 
 .. index:: ! string, ! bytes
 
-.. _strings:
+.. _string:
```
Comment (chriseth) [inline docs/types/reference-types.rst]: I think this should not be changed.

Comment (axic) [reply docs/types/reference-types.rst]: Why? I think it may make sense changing, if all the references are changed too of course :)

Comment (leonardoalt) [reply docs/types/reference-types.rst]: @FeiyangTan If this is changed the references need to be changed too.

Comment (chriseth) [inline docs/types/reference-types.rst]:
```suggestion
.. _strings:
```

**Review by chriseth — APPROVED** [review]

**Pull request closed without merging**